portfolio/src/main/webapp/script2.js

```diff
@@ -47,11 +47,16 @@ function addRandomFact() {
 function tabulateAnswers() {
   // initialize variables for each choice's score
   // If you add more choices and outcomes, you must add another variable here.
-  //*******
-  cvals = ['c1','c2','c3','c4'];
-  cscores = [0,0,0,0];
   
+  //*******
+  //cvals = ['c1','c2','c3','c4'];
+  //cscores = [0,0,0,0];
+  var cdict = {'c1': 0,
```

Comment:
I would advise naming your variables to be more descriptive of what they contain.  Not so much of a problem now since 'c' is understood to be 'choice' throughout the file.  But once you get to bigger files, it'll get hard to keep track of what 'c' means.  Something like 'choices_dict' or 'choices_score' would work well here.